addon/components/bs-modal/footer.hbs

```diff
@@ -9,14 +9,15 @@
       }}
         {{bs-default @closeTitle "Ok"}}
       {{/component}}
-      {{#component
-        (bs-default @buttonComponent (component "bs-button"))
-        type=(bs-default @submitButtonType "primary")
-        onClick=@onSubmit
-        _disabled=(bs-default @submitDisabled false)
-      }}
+      {{#let (bs-default @buttonComponent (component "bs-button")) as |Button|}}
+        <Button
+          @type={{bs-default @submitButtonType "primary"}}
+          @onClick={{@onSubmit}}
+          disabled={{bs-default @submitDisabled false}}
+        >
```

Comment:
Awesome that this pattern works! That means we can drop `_disabled` and similar arguments only kept for internal usage! :tada:

Reply:
Not quite I think: that works only when we invoke (i.e. render) the component, but not when yielding it back. Which we do...